Dockerfile-DOI

```diff
@@ -27,7 +28,8 @@ RUN rm -f /etc/apt/sources.list.d/chris-lea-node_js-jessie.list \
 && curl --cacert DOIRootCA2.cer https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add - \
 && echo 'deb https://deb.nodesource.com/node_6.x jessie main' > /etc/apt/sources.list.d/nodesource.list \
 && apt-get update \
-&& apt-get install -y nodejs
+&& apt-get install -y nodejs \
+&& apt-get install -y libfontconfig
```

Comment:
Same as in the other dockerfile: I think this should be at the top of the file, in its own command, or included with a more related command